static_precompiler/utils.py

```diff
@@ -130,26 +130,32 @@ def run_command(args, input=None, cwd=None):
 
 class URLConverter(object):
 
-    URL_PATTERN = re.compile(r"url\(([^\)]+)\)")
+    URL_PATTERN = re.compile(r"url\(([^\)]+)\);")
 
     @staticmethod
     def convert_url(url, source_dir):
         assert source_dir[-1] == "/"
-        url = url.strip(' \'"')
+
         if url.startswith(('http://', 'https://', '/', 'data:')):
             return url
-        return urljoin(settings.STATIC_URL, urljoin(source_dir, url))
+        return '{0}'.format(urljoin(settings.STATIC_URL, urljoin(source_dir, url)))
```

Comment:
Why?

Reply:
Oh sorry, that's a leftover from a previous try. I'm removing it right now